lib/school.rb

```diff
@@ -0,0 +1,53 @@
+class School
+  attr_reader :start_time, :hours_in_school_day, :student_names
+  def initialize(start_time, hours_in_school_day)
+    @start_time = start_time
+    @hours_in_school_day = hours_in_school_day
+    @student_names = []
+    
+  end
+
+  def add_student_name(name)
+    @student_names << name
+  end
+
+  def end_time
+    int_start_time = @start_time.to_i
```

Comment:
Nitpick: You shouldn't abbreviate words as it can confuse other developers that have to read your code.  Take the time to spell out `initial` here for developer empathy ❤️